docs/pages/workflow/glossary-of-terms.md

```diff
@@ -33,60 +129,177 @@ The term "eject" was popularized by [create-react-app](https://github.com/facebo
+
 ### Experience
 
 A synonym for app that usually implies something more single-use and smaller in scope, sometimes artistic and whimsical.
 
+### Expo Autolinking
+
+The original [Autolinking](#autolinking) system that is designed for project's using `expo-modules-core`. This system links modules based on the existence of a `expo-module.config.json` file in the library's root directory.
+
 ### Expo CLI
 
-The command-line tool for working with Expo. [Read more](expo-cli.md).
+The command-line tool for working with Expo. [Read more](expo-cli.md). This term now refers to the [Local Expo CLI](#local-expo-cli), but historically referred to the [Global Expo CLI](#global-expo-cli).
+
+### Expo client
+
+The former name for the [Expo Go](#expo-go) app.
+
+### Expo Config
+
+A file named `app.json`, `app.config.json`, `app.config.js`, or `app.config.ts` in the root of a project directory. [Learn more](configuration.md).
+
+This file is used for the following purposes:
+
+- Configuring how [Expo CLI](#expo-cli) works.
+- Generating a project's public [manifest](#manifest) in EAS Update (think index.html but for native apps).
+- Adding Expo [Config Plugins](#config-plugin) and configuring how `npx expo prebuild` generates native code.
+
+### Expo Export
+
+Refers to the command `npx expo export` from [Expo CLI](#expo-cli). This command is used to bundle the application JavaScript and assets, then export them into a static folder that can be uploaded to a hosting service like [EAS Update](#eas-update), and embedded in a [native runtime](#native-runtime) for offline use.
 
 ### Expo Go
 
 The iOS and Android app that runs Expo apps. When you want to run your app outside of the Expo Go app and deploy it to the App and/or Play stores, you can build a [Standalone App](#standalone-app).
 
-### Expo client
+### Expo Install
 
-The former name for the [Expo Go](#expo-go) app.
+Refers to the command `npx expo install` from [Expo CLI](#expo-cli). This command is used to install NPM packages containing [native modules](#native-module) that work with the currently installed version of `expo` in the project. Not all packages are supported. This command wraps the globally installed [package managers](#package-manager).
+
+### Expo Module Config
+
+A file named `expo-module.config.json` that lives in the root directory of a [native module](#native-module). [Learn more](/modules/module-config/).
 
 ### Expo SDK
 
 The Expo SDK provides access to device/system functionality such as camera, push notification, contacts, file system, and more. Scroll to the SDK API reference in the documentation navigation to see a full list of APIs and to explore them. [Read more about the Expo SDK](/versions/latest/). [Find it on GitHub](https://github.com/expo/expo-sdk).
```

Comment:
```suggestion
The Expo SDK provides access to device/system functionality such as camera, push notification, contacts, file system, and more. Scroll to the SDK API reference in the documentation navigation to see a full list of APIs and to explore them. [Read more about the Expo SDK](/versions/latest/). [Find it on GitHub](https://github.com/expo/expo-sdk).
```

Hmm, linked repository https://github.com/expo/expo-sdk do not exist. Also probably we could write it a bit better, to avoid two link length sentences at the end.

Reply:
haha this is a frustratingly old doc

Reply:
So it's pre-monorepo times? 😄

Reply:
Updated with a more modern description.